Gemfile

```diff
@@ -51,7 +51,7 @@ gem 'rqrcode'
 gem 'ruby-progressbar'
 gem 'ruby-saml'
 gem 'safe_target_blank', '>= 1.0.2'
-gem 'saml_idp', git: 'https://github.com/18F/saml_idp.git', tag: '0.11.0.18f'
+gem 'saml_idp', git: 'https://github.com/18F/saml_idp.git', branch: 'stages/orenyk' #tag: '0.11.0.18f'
```

Comment:
Need to cut a tag and update this when https://github.com/18F/saml_idp/pull/36 is approved/merged